scripts/check

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
-set -e
+#!/bin/bash
+set -Eeou pipefail
 
-npx tsc --noEmit && npx svelte-check
+npx tsc --noEmit
+npx svelte-check --tsconfig tsconfig.json
```

Comment:
Nice find!

After going through the other configuration options I'm thinking maybe we should also enable `--use-new-transformation` ? It catches a few more type issues in `Profile.svelte`

Reply:
I thought about this and tried enabling it, but couldn't find a quick fix for the problems it found. Would rather leave it for either a follow-up or wait until they turn this feature on by default once it's not experimental anymore.